AccountService/src/main/java/com/account/entity/Account.java

```diff
@@ -0,0 +1,50 @@
+package com.account.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Data;
+
+@Entity
```

Comment:
if will not work, please try to add lombok annotations

Reply:
I have used @Data annotation of lombok which as setter, getter and constructor annotations in it